<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaListarLocacao</class>
 <widget class="QDialog" name="TelaListarLocacao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_Voltar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblLocacao">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>651</width>
     <height>331</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Alterar">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>560</y>
     <width>126</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Devolver Veiculo</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>430</y>
     <width>174</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Codigo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtNomeCodigo">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>1079</y>
     <width>601</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Fazer Locação</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Fazer Devolução</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Listar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>460</y>
     <width>182</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_14">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>CLIENTE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtNomeCliente">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>490</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>MOTORISTA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtNomeMotorista">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>430</y>
     <width>184</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>VEICULO</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="TxtVeiculo">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>460</y>
     <width>263</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>QUILOMETRAGEM FINAL </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtQuilFinal"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>500</y>
     <width>238</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>VALOR DO CAUÇÃO</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtCaucao"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>500</y>
     <width>239</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>VALOR DO SEGURO </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtSeguro"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>440</y>
     <width>235</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>PAGAMENTO FINAL</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtPagamentoFinal"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
